provisioning/e2e/_provisioning_e2e.js

```diff
@@ -359,6 +434,10 @@ describe('IoT Provisioning', function() {
       testObj: new TpmIndividual()
     },
     */
+    {
+      testName: 'Symmetric Key individual enrollment',
+      testObj: new SymmetricKeyIndividual()
```

Comment:
>SymmetricKeyIndividual [](start = 19, length = 22)

isnt' symmetric key also available for group enrollment? we should test this too

Reply:
Basically the only difference between the two is that for the symmetric key parameter we append the registrationId to the group primary key and hmac it.  Pass that as the symmetric key.  No other changes.  Now I could show that particular operation in a separate sample and/or add it to this e2e test.  C thinks this should be in either.  I am ambivalent.

---
In reply to: [231363456](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r231363456) [](ancestors = 231363456)

Reply:
Done.

---
In reply to: [232118504](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r232118504) [](ancestors = 232118504,231998518,231363456)